src/script/view_model/WarningsContainer.tsx

```diff
@@ -385,22 +381,30 @@ const WarningsContainer: React.FC = () => {
               {t('warningLifecycleUpdateNotes')}
             </a>
             &nbsp;·&nbsp;
-            <a
-              className="warning-bar-link"
+            <button
+              className="warning-bar-link button-reset-default"
               onClick={() => amplify.publish(lifeCycleRefresh)}
-              rel="nofollow noopener noreferrer"
-              target="_blank"
               data-uie-name="do-update"
             >
               {t('warningLifecycleUpdateLink')}
-            </a>
+            </button>
           </div>
         </div>
       )}
     </div>
   );
 };
 
+interface CloseWarningButtonProps {
+  onClick: () => void;
+}
+const CloseWarningButton = ({onClick}: CloseWarningButtonProps) => (
```

Comment:
Can this component live here or should I move it to separate file?

Reply:
I don't even think we need a component for this, we could store it in a variable at the top of the `WarningContainer` component and already bind it to the `close` function. 
Like so

```ts
const closeButton = (  <button
    type="button"
    data-uie-name="do-close-warning"
    className="warning-bar-close icon-close button-round button-round-dark button-reset-default"
    onClick={closeWarning}
  />)
```

and then in the render just use that var

```ts
<div>{closeButton}</div>
```

WDYT?

Reply:
because this way it gets recreated on every render

Reply:
sure, if is was bigger with some logic we would def move it to an external component :ok_hand: